### ISSUE TITLE
window.testState.pickleStep doesn't hold currently executing step information

### ISSUE THREAD

**Issue by WTK** [badeball/cypress-cucumber-preprocessor#836]
### Current behavior

When attempting to use (for example log) current step text using `window.testState.pickleStep.text` the resulting item contains value of the last step within current scenario. See test code to reproduce section below, currently the text printed would be `the "John Doe" text is displayed` while executing `the admin logs in` step.

### Desired behavior

`window.testState.pickleStep.text` should hold text of currently executing step.

### Test code to reproduce

```
# feature file
Feature: Joining the list

    Scenario: Join the list
        When the admin logs in
        Then the "John Doe" text is displayed

# step definition
When('the admin logs in', () => {
  cy.log(window.testState.pickleStep.text);
});
```

### Versions

* **Cypress version**: 10.8
* **Preprocessor version**: 13.0.1
* **Node version**: 14.14.0

### Checklist

- [x] I've read the [FAQ](https://github.com/badeball/cypress-cucumber-preprocessor/blob/master/docs/faq.md).
- [x] I've read [Instructions for logging issues](https://github.com/badeball/cypress-cucumber-preprocessor/blob/master/CONTRIBUTING.md#instructions-for-logging-issues).
- [x] I'm not using `cypress-cucumber-preprocessor@4.3.1` (package name has changed and it is no longer the most recent version, see [#689](https://github.com/badeball/cypress-cucumber-preprocessor/issues/689)).

**Comment by WTK** [comment]
@badeball Since you pushed the fix for this, any chance you could publish new version?

**Comment by badeball** [comment]
Yeah yeah, hold your horses.

**Comment by WTK** [comment]
I'm anxiously waiting as we have ci failing because of it 😉

**Comment by badeball** [comment]
Released in v13.0.2.
